snippets/markdown.json

```diff
@@ -69,14 +69,14 @@
 		"description": "Alert box with choice of info, danger, or warning"
 	},
 
-	"Beta callout":{
-		"prefix": ";;beta",
+	"Callout":{
+		"prefix": ";;callout",
 		"body": [
-			"{{< beta-callout url=\"https://google.com\" d-toggle=\"modal\" d_target=\"#signupModal\" custom_class=\"sign-up-trigger\" btn_hidden=\"false\">}}",
+			"{{< callout url=\"https://google.com\" d-toggle=\"modal\" d_target=\"#signupModal\" custom_class=\"sign-up-trigger\" btn_hidden=\"false\">}}",
```

Comment:
`d-toggle=\"modal\"` should be `d_toggle` with an underscore

Reply:
Thanks @StefonSimmons, good catch!